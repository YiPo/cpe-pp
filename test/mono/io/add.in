2 3
12 19
53 81
