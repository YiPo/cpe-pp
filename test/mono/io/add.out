5
31
134
